client/my-sites/email/inbox/index.jsx

```diff
@@ -89,7 +89,7 @@ const hasAtLeastOneMailbox = ( domains ) =>
 const showActiveDomainList = ( domains ) =>
 	domains.some( ( domain ) => hasPaidEmailWithUs( domain ) || hasEmailForwards( domain ) );
 
-const InboxManagement = () => {
+const InboxManagement = ( { selectedIntervalLength, source } ) => {
```

Comment:
I don't think we should pull the `source` from the URL -- if we're viewing `InboxManagement`, then the value of the `source` prop in `EmailManagementHome` should _always_ be `INBOX_SOURCE`.

Reply:
Apologies if I'm missing something but as far I can see we still set it to `INBOX_SOURCE`. However we do that in the controller rather than in the component itself.

Reply:
🤦 You are 100% right, @stephanethomas!

So this is not blocking, but I don't know that it's worth pushing this out to the controller as we don't have any use cases where it might be a different value.

Reply:
Ok sorry, I tried to anticipate to future possible use cases. I'll revert the source parameter to be handled as it was done before.